automated_testing/test/pageobjects/index-page.ts

```diff
@@ -112,7 +108,7 @@ class IndexPage extends Page {
     }
 
     public get firstCompeteCardButtonActiveContests() {
-        return $(selectors.pageAllActiveCardsSelector).$(selectors.pageCompeteButtonIdSelector);
+        return $(selectors.pageAllActiveCardsSelector).$$(selectors.pageCompeteButtonIdSelector)[0];
```

Comment:
Isn't this the same logic as in `competeButtonInActiveContestCard()`?
Maybe just do `competeButtonInActiveContestCard()[0]`